### PR TITLE
C extend

### PR DESCRIPTION
**Pull request by rogiervandam**
## Description
New improved algorithm in C. Should be significantly faster. 

## Contributing requirements

* [x] I read the contribution guidelines in CONTRIBUTING.md.
* [x] I placed my solution in the correct solution folder.
* [x] I added a README.md with the right badge(s).
* [x] I added a Dockerfile that builds and runs my solution.
* [x] I selected `drag-race` as the target branch.
* [x] All code herein is licensed compatible with BSD-3.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.vs
+.vscode
```
Comment (rbergen) [inline .gitignore]: Please remove this top-level .gitignore from the PR's commits

**Review by rogiervandam — COMMENTED** [review]
Removed .gitignores

**Comment by rbergen** [comment]
LGTM. I'm requesting a second review because it concerns a new solution.

**Review by rbergen — APPROVED** [review]

**Review by marghidanu — APPROVED** [review]
LGTM

**Pull request merged**